docs/guide/basic-config.md

```diff
@@ -5,12 +5,12 @@
 Without any configuration, the page is pretty minimal, and the user has no way to navigate around the site. To customize your site, let's first create a `.vuepress` directory inside your docs directory. This is where all VuePress-specific files will be placed in. Your project structure is probably like this:
 
 ```
-cwd
-├─── docs
-│   ├── README.md
-│   └── .vuepress
-│       └── config.js
-└── package.json
+.
+├─ docs
+│  ├─ README.md
+│  └─ .vuepress
+│     └─ config.js
+└─ package.json
```

Comment:
`dot` here would have the different meaning with comparing to other parts. we should keep the `cwd`